src/core/modules/Planet.cpp

```diff
@@ -777,6 +789,7 @@ QVariantMap Planet::getInfoMap(const StelCore *core) const
 			map.insert("eclipse-obscuration", eclipseObscuration);
 			if (core->getCurrentPlanet()==ssystem->getEarth())
 			{
+				// FIXME: This is for Solar eclipses only. Make sure it works for Mercury/Venus transits, or omit it then!
```

Comment:
Why FIXME here? This code made exactly for Solar eclipses on Earth only. Or you want to compute "Eclipse magnitude" for transits?

Reply:
OK, I have an idea how to resolve this issue

Reply:
OK, I've fixed it in master